code/game/objects/items/storage/storage.dm

```diff
@@ -444,6 +444,11 @@
 			H.put_in_hands(src)
 			H.r_store = null
 			return
+		if(block_open_while_equipped && slot_flags && (src in H.get_all_slots()))

```

Comment:
Knock that whole thing down a tab and do:

`if(block_open_while_equipped && slot_flats && C.back == src)`

after casting C as a carbon. Monkeys aren't humans on /tg/, but carbons. They also get backpacks.

Reply:
Bad idea. Did this specifically so that the behavior wasn't tied to the back slot because this is defined on the `/obj/item/storage` level, and people may want to implement this behavior for some storage item on a slot that isn't the back. Like syndicate spy shoes that have a storage slot, but must be removed to be accessed or some nonsense.

Reply:
Could put a check below to see if the mob is of the type monkey or an relevant subtypes, but that seems a bit ugly.